File: use_crypto.py
from java import type
DummyEncryptor = type('com.example.crypto.DummyEncryptor')

msg = "Grantly Rocks!"
enc = DummyEncryptor.encrypt(msg)
dec = DummyEncryptor.decrypt(enc)

print("Original :", msg)
print("Encrypted:", enc)
print("Decrypted:", dec)

File: useCrypto.js
const DummyEncryptor = Java.type("com.example.crypto.DummyEncryptor");

const msg = "Grantly Rocks!";
const enc = DummyEncryptor.encrypt(msg);
const dec = DummyEncryptor.decrypt(enc);

console.log("Original :", msg);
console.log("Encrypted:", enc);
console.log("Decrypted:", dec);
